src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/Set-AzConnectedKubernetes.ps1

```diff
@@ -88,6 +88,8 @@ function Set-AzConnectedKubernetes {
     param(
         [Parameter(ParameterSetName = 'SetExpanded', Mandatory)]
         [Parameter(ParameterSetName = 'SetExpandedDisableGateway', Mandatory)]
+        [Parameter(ParameterSetName = 'SetExpandedEnableWorkloadIdentity', Mandatory)]
+        [Parameter(ParameterSetName = 'SetExpandedDisableWorkloadIdentity', Mandatory)]
```

Comment:
Why are these separate parameter sets? I'm not too familiar, but as I understand them, parameter sets are for defining different ways of calling the cmdlet. So for example, I'm not surprised we have these two:
- `Set` (i.e. `Set-AzConnectedKubernetes -InputObject $cluster`)
- `SetExpanded` (i.e. `Set-AzConnectedKubernetes -Name blah -ResourceGroup -blah ...`)

but I'm a bit surprised we need separate ones for enabling / disabling gateway and workload identity

is it because `-WorkloadIdentityEnabled` and `-WorkloadIdentityDisabled` are mutually exclusive?